back-end/admin-page/src/app/App.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { BrowserRouter, Switch, Route, Redirect } from "react-router-dom";
+import { Login, Lab } from "../components";
+
+function App({ apiService }) {
```

Comment:
```suggestion
const App = ({ apiService }) => {
```

function 은 악마입니다 :imp:

Reply:
왜 악마인가요? 😈 수정했어요